R/HASPT.R

```diff
@@ -88,17 +83,31 @@ HASPT = function(angle, sptblocksize = 30, spt_max_gap = 60, ws3 = 5,
       outofspt = c(0,outofspt,0)
       s3 = which(diff(outofspt) == 1) #start of blocks out of spt?
       e3 = which(diff(outofspt) == -1) #end blocks out of spt?
+      # starting gap not to be filled
```

Comment:
If, for example, first `nomov` segment is found at 12:30, then there is a first block `outofspt` from 12:00 to 12:30. Since this block is shorter than 60 min, it gets filled and considered to be part of the SPT when it should not. 
This should then be "starting block not to be filled" instead of "gap", right?

Reply:
Yes, correct. I will now update the branch from master and edit the comments.